arangod/RestServer/InitDatabaseFeature.cpp

```diff
@@ -123,7 +125,7 @@ std::string InitDatabaseFeature::readPassword(std::string const& message) {
 
   arangodb::Logger::flush();
   // Wait for the logger thread to flush eventually existing output.
-  sleep(0.5);
+  std::this_thread::sleep_for(std::chrono::microseconds(500));
```

Comment:
@dothebart if this should be 1/2 secs use `std::chrono::milliseconds` microsecond is 1 / 10^6, millisecond 1 / 1000.